content/case_histories/hoop_region_norway/processinginterpretation.rst

```diff
@@ -0,0 +1,207 @@
+.. _hoop_region_norway_processinginterpretation:
+
+Processing and Interpretation
+=============================
+
+CSEM Data
+---------
+
+In order to resolve the interpretation ambiguity inherent when only seismic data are considered, we used resistivity information derived from controlled source electromagnetic (CSEM) data. CSEM has been used successfully as an offshore hydrocarbon exploration tool for over a decade (:cite:`Constable2010`). The data for this study has been collected in a relatively new way; using a towed source and array of electric field receivers (:cite:`Englemark2014,Key2014`). MacGregor and Tomlinson (:cite:`MacGregor2014`) provides an overview to the offshore CSEM method and the various acquisition methods that can be applied.
+
+.. (Constable 2010)
```

Comment:
Those are a reminder of what I am citing. I want to keep them there for the time being.